test/oss_test.cpp

```diff
@@ -20,120 +20,172 @@
 #include <unistd.h>	
 #include <gtest/gtest.h>
 
+using namespace emeralddb::oss;
+using namespace std;
+using namespace testing;
+
+
+class ossSocketTest : public Test {
+public:
+  ossSocketTest() {}
+  ~ossSocketTest() {}
+
+protected: 
+  virtual void SetUp() {
```

Comment:
here the SetUp/TearDown two virtual methods that are both decorated by protected are two methods inherit from gtest's class Test. we can override it by add more context information however cannot modify their methods' signature